.github/workflows/publish_bundle.yaml

```diff
@@ -1,6 +1,11 @@
 name: Release Bundle
 
 on:
+  push:
+    branches:
+      - main
+    paths:
+      - "bundle/**"
   workflow_dispatch:
 
 run-name: Release bundle by @${{ github.actor }}
```

Comment:
Lower down, this action needs to be fixed to use the upload-bundle action instead of upload-charm.

Reply:
But we have this comment in the file:

```
# Note the use of the upload-charm action. Bundles can be treated as charms.
```

Do you think we've used `upload-charm` intentionally here?

Reply:
I agree it's best to use `upload-bundle`.

Reply:
> Do you think we've used upload-charm intentionally here?

I added that, and it can be treated as a charm by the `charmcraft` CLI tool but the action does things differently so this statement was wrong. Sorry I should've fixed it before.